mentat/llm_api_handler.py

```diff
@@ -238,3 +240,13 @@ async def is_model_available(self, model: str) -> bool:
             model.id async for model in self.async_client.models.list()
         ]
         return model in available_models
+
+    async def call_whisper_api(self, audio_path: Path) -> str:
+        raise_if_in_test_environment()
+
+        audio_file = open(audio_path, "rb")
+        transcript = self.client.audio.transcriptions.create(
```

Comment:
Is there a reason why we shouldn't asynchronously send the user's voice instead of using the synchronous client?